aws-datastore/src/test/java/com/amplifyframework/datastore/AWSDataStorePluginTest.java

```diff
@@ -204,7 +225,7 @@ public void configureAndInitializeInApiModeWithoutApi() throws JSONException, Am
      * @throws AmplifyException on failure to arrange API plugin via Amplify facade
      */
     @Test
-    public void clearStopsSyncUntilNextInteraction() throws AmplifyException, JSONException {
+    public void clearStopsSyncAndDeletesDatabase() throws AmplifyException, JSONException {
```

Comment:
Line 223 still says "synchronization processes are restarted"; could you update the Javadoc?

Reply:
The comment was actually wrong before - the previous implementation restarted synchronization immediately).   

The new behavior matches the comment now.  `clear` won't restart synchronization, but the next interaction with DataStore will.